src/lucky/web_socket.cr

```diff
@@ -0,0 +1,8 @@
+module Lucky
+  class WebSocket < HTTP::WebSocketHandler
+    getter proc
+
```

Comment:
This class is probably not necessary, but I figure it makes it easier to reference a Lucky namespace, than digging around to see what the built-in crystal one is. Plus we could do some delegation stuff.